ios/CodePush/CodePushDownloadHandler.m

```diff
@@ -22,12 +24,12 @@ -(void)download:(NSString*)url {
     NSURLRequest *request = [NSURLRequest requestWithURL:[NSURL URLWithString:url]
                                              cachePolicy:NSURLRequestUseProtocolCachePolicy
                                          timeoutInterval:60.0];
-    
+    NSOperationQueue *delegateQueue = [NSOperationQueue new];
```

Comment:
Would it make more sense to move this above the `[connection setDelegateQueue]` call?